scrapper.py

```diff
@@ -26,6 +27,8 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
         pass
```

Comment:
remove pass if you implement the function - it is a simple stub that should be removed